ccm_web/client/src/index.tsx

```diff
@@ -1,19 +1,25 @@
 import { SnackbarProvider } from 'notistack'
 import React from 'react'
 import ReactDOM from 'react-dom'
-import { BrowserRouter } from 'react-router-dom'
+import { BrowserRouter, Route, Switch } from 'react-router-dom'
 import { ThemeProvider } from '@material-ui/core'
 
 import ccmTheme from './theme'
 import App from './App'
+import AccessDenied from './pages/AccessDenied'
 import './index.css'
 
 ReactDOM.render(
   <React.StrictMode>
    <ThemeProvider theme={ccmTheme}>
     <SnackbarProvider maxSnack={3}>
       <BrowserRouter>
-        <App />
+        <Switch>
+          <Route exact={true} path='/access-denied'>
+            <AccessDenied email='4help@umich.edu' helpLink='https://its.umich.edu/help' />
```

Comment:
Is it okay to have these values be hard-coded? We could put them in the configuration, but I'm not sure how we would actually send to them to the client, since the user isn't authenticated and doesn't have access to `globals`. We'd need a public API endpoint with public data, but maybe that's more of a hassle than it's worth. Maybe it's possible to put the data in `index.html` and grab it (like we do in our Django projects), but we don't have a mechanism for putting configuration variables in that file yet.

Reply:
Or maybe something like this `DefinePlugin` would work for setting a couple public variables. https://webpack.js.org/plugins/define-plugin/

Reply:
Let me know what you think. I can work on this now or as a separate issue. `DefinePlugin` seems reasonable, but would have to try using `config.ts` in the `webpack` artifacts.

Reply:
I feel it is ok to keep this hard-coded until we see a possible collaboration with other schools interested in using CCM for their schools. Since lot of ourText still has `U-M` in the UI. Hope this will convince you not to worry about this now. You can create an issue for later, definitely something interesting to solve how to load static links based on configuration.